### ISSUE TITLE
email validation returning false in live server

### ISSUE THREAD

**Issue by muralibobby35** [tintnaingwinn/email-checker#3]
Hi,

We are using this package. it is working well in local and our test and stage domains. But one of our client email not verifying in live server even it was exist. If we check same email in local and test and stage servers it is returning true.

What will be the problem? could you please help on this

**Comment by tintnaingwin** [comment]
Hello @muralibobby35 , Let me know your client email.

**Comment by muralibobby35** [comment]
My apologies, i can't share it because its of our office rules.

I have checked with other email validator tools which is in online

Checked in EmailHippo: https://tools.verifyemailaddress.io
Result | Unverifiable

Reason | ServerIsCatchAll
-- | --

Trust score (0-10) | 3
-- | --

Hosting provider | Office365
-- | --


I have checked in https://verifalia.com/validate-email

I got below response:  Risky email type, the external mail exchanger accepts fake and non-existent e-mail addresses; therefore the provided e-mail address MAY not exist. The existence of the individual mailbox cannot be verified.

**Classification**: Risky
**Status**: Failed to validate, the external mail exchanger rejected the local end point. Retrying the validation may result in a definitive response.
**Status code:** LocalEndPointRejected 


I have checked again got this below response:

**Classification**: Risky
**Status**: Risky email type, the external mail exchanger accepts fake and non-existent e-mail addresses; therefore the provided e-mail address MAY not exist. The existence of the individual mailbox cannot be verified.
**Status code**: ServerIsCatchAll
